packages/host/tests/acceptance/code-submode/file-tree-test.ts

```diff
@@ -592,6 +593,39 @@ module('Acceptance | code submode tests', function (hooks) {
       'expected near-bottom file to be visible after opening it',
     );
   });
+
+  test('can open files in base realm', async function (assert) {
+    let codeModeStateParam = stringify({
+      stacks: [
+        [
+          {
+            id: `${testRealmURL}Person/1`,
+            format: 'isolated',
+          },
+        ],
+      ],
+      submode: 'code',
+      fileView: 'browser',
+      codePath: `http://localhost:4201/base/cards-grid.gts`,
```

Comment:
I suppose this should be formed with `testRealmURL`? 

Also can you include a screenshot with before/after, that would help to see what was fixed

Reply:
This issue is specific to the base realm because we map the URL for the base realm `cardstack.com` to `localhost:4201`. That is why, in this test, I use `localhost:4201`.

Before: 
![screencast 2023-10-24 14-34-48](https://github.com/cardstack/boxel/assets/12637010/d8b8403b-a630-42a1-9524-aedc8c21e70d)

After:

https://github.com/cardstack/boxel/assets/12637010/61aea8f1-1eae-4f5e-ac14-6d6d8fee67cd